docs/quick-ingestion-guides/looker/configuration.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Configuration
+---
+# Configuring Your Looker Connector to DataHub
+
+Now that you have created a DataHub specific API key with the relevant access in [the prior step](setup.md), it's now time to set up a connection via the DataHub UI.
+
+## Configure Secrets
+
+1. Within DataHub, navigate to the **Ingestion** tab in the top, right corner of your screen
+
+<p align="center">
+  <img width="75%" alt="Navigate to the &quot;Ingestion Tab&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_ingestion_button.png"/>
+</p>
+
+:::note
+If you do not see the Ingestion tab, please contact your DataHub admin to grant you the correct permissions
+:::
+
+2. Navigate to the **Secrets** tab and click **Create new secret**.
+
+<p align="center">
+   <img width="75%" alt="Secrets Tab" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_secrets_tab.png"/>
+</p>
+
+3. Create a client id secret
+
+  This will securely store your Looker `API Key Client ID` within DataHub
+
+   * Enter a name like `LOOKER_CLIENT_ID` - we will use this later to refer to the `API Key Client ID`
+   * Enter the `API Key Client ID`
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="API Key Client ID" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-client-id-secret.png"/>
+</p>
+
+4. Create a secret to store the API Key Client Secret
+
+  This will securely store your client secret"
+
+   * Enter a name like `LOOKER_CLIENT_SECRET` - we will use this later to refer to the client secret
+   * Enter the client secret
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="API Key client secret" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-client-secret.png"/>
+</p>
+
+## Configure Recipe
+1.  Navigate to the **Sources** tab and click **Create new source**
+
+  <p align="center">
+    <img width="75%" alt="Click &quot;Create new source&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_click_create_new_source_button.png"/>
+  </p>
+
+2.  Choose Looker
+
+  <p align="center">
+    <img width="70%" alt="Select Looker from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-choose-looker.png"/>
```

Comment:
the image is broken

Reply:
@yoonhyejin 
We need to merge this static PR first: https://github.com/datahub-project/static-assets/pull/37

Reply:
merged that PR